static/app/components/updatedEmptyState.tsx

```diff
@@ -43,32 +28,26 @@ export default function UpdatedEmptyState({project}: {project?: Project}) {
     p => p.id === currentPlatformKey
   ) as PlatformIntegration;
 
-  const platformPath = getPlatformPath(currentPlatform);
-
-  const module = useLoadGettingStarted({
-    platformId: currentPlatform.id,
-    platformPath,
+  const loadGettingStarted = useLoadGettingStarted({
```

Comment:
same destructuring idea could be done here too!

Reply:
In this case, I prefer to leave it as it is because we use multiple hooks with `isLoading`, etc in this file. I believe it's better to maintain the current approach rather than using aliases